### PR TITLE
docs: first steps

### PR DESCRIPTION
**Pull request by kostis-codefresh**
Signed-off-by: Kostis Kapelonis <kostis@codefresh.io>

Ongoing work for https://github.com/argoproj/argo-cd/issues/5635

Checklist:

* [X] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [X] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [X] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [X] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

File: docs/getting_started/first_steps.md [hunk 1]
```diff
@@ -1,11 +1,184 @@
...
+
+!!! note
+    The CLI environment must be able to communicate with the Argo CD API server. If it isn't directly accessible as described in the previous section, you can tell the CLI to access it using port forwarding through one of these mechanisms: 1) add `--port-forward-namespace argocd` flag to every CLI command; or 2) set `ARGOCD_OPTS` environment variable: `export ARGOCD_OPTS='--port-forward-namespace argocd'`.
+
+You can change the admin password using the command:
+
+```bash
+argocd account update-password
+```
+
```
Comment (jannfis) [inline docs/getting_started/first_steps.md:92]: Maybe add an additional paragraph to state something like "In order to change the password of the admin user, you will need to know the current password. If you don't know the current password, you will have to use a different method to reset it."

File: docs/getting_started/first_steps.md [hunk 2]
```diff
@@ -1,11 +1,184 @@
...
+The application status is initially in `OutOfSync` state since the application has yet to be
+deployed, and no Kubernetes resources have been created. To [sync](../syncing/index.md) (deploy) the application, run:
+
+```bash
+argocd app sync guestbook
+```
+
+This command retrieves the manifests from the [repository](../basics/repos/index.md) and performs a `kubectl apply` of the
+manifests. The guestbook app is now running and you can now view its resource components, logs,
```
Comment (jannfis) [inline docs/getting_started/first_steps.md:131]:
```suggestion
This command retrieves the manifests from the [repository](../basics/repos/index.md) and performs a `kubectl apply` on the cluster. The guestbook app is now running and you can now view its resource components, logs,
```

**Review by jannfis — COMMENTED** [review]
Great stuff, @kostis-codefresh! I like that easy walk-through very much, instructions are clear.

I have only some nits, please see below.

**Review by terrytangyuan — APPROVED** [review]
Looks great!

**Review by jannfis — APPROVED** [review]
LGTM! Thanks a lot again @kostis-codefresh

**Pull request merged**